engine/resources/cron.go

```diff
@@ -80,9 +81,18 @@ type CronRes struct {
 
 	init *engine.Init
 
+	// Unit is the name of the systemd service unit. It is only necessary to
```

Comment:
You mixed up your commits. This change shouldn't be in this commit, etc...

Reply:
I meant to rebase that change to the initial commit, but by the time I'd finished breaking up the session commit, it slipped my mind. Now it's fixed.